docs/proposals/20200330-spot-instances.md

```diff
@@ -214,7 +214,7 @@ when the instance is created:
   (Delete is supported for VMs as part of VMSS only).
 
 - BillingProfile (default: -1) : This is a struct containing a single field, `MaxPrice`.
-  This is a float representation of the maximum price the user wishes to pay for their VM.
+  This is a string representation of the maximum price the user wishes to pay for their VM.
```

Comment:
Can you please add a small comment about why we're using a string for the record? It might not be obvious to future readers.